README.md

```diff
@@ -109,36 +109,11 @@ mv /path/to/directory/role.prom.$$ /path/to/directory/role.prom
 
 ### Filtering enabled collectors
 
-The node_exporter will expose all metrics from enabled collectors by default, but it can be passed an optional list of collectors to filter metrics. The `collect[]` parameter accepts values matching enabled collector names.
-
-This can be useful for specifying different scrape intervals for different collectors in Prometheus:
-
-```yaml
-scrape_configs:
-  - job_name: 'node resources'
-    scrape_interval: 15s
-    static_configs:
-      - targets:
-        - '192.168.1.2:9100'
-    params:
-      collect[]:
-        - cpu
-        - meminfo
-        - diskstats
-        - netdev
-        - netstat
-
-  - job_name: 'node storage'
-    scrape_interval: 1m
-    static_configs:
-      - targets:
-        - '192.168.1.2:9100'
-    params:
-      collect[]:
-        - filefd
-        - filesystem
-        - xfs
-```
+The `node_exporter` will expose all metrics from enabled collectors by default.  This is the recommended way to collect metrics to avoid errors when comparing metrics of different familes.
+
+For advanced use the `node_exporter` can be passed an optional list of collectors to filter metrics. The `collect[]` parameter accepts values matching enabled collector names.
```

Comment:
It's not quite clear that this parameter must be provided multiple times for multiple names, or that this happens on /metrics. Maybe an example URL?

Reply:
I'll include a partial example of a scrape_config.